website/integrations/services/whats-up-docker/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: What's Up Docker
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is What's Up Docker
+
+> WUD is an easy-to-use tool that alerts you whenever a new version of your Docker containers is released.
+>
+> -- https://fmartinou.github.io/whats-up-docker/
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `wud.company` is the FQDN of the WUD install.
+-   `authentik.company` is the FQDN of the authentik install.
+
+## WUD configuration
+
+To configure WUD to use authentik, add the following values to your `.env` file:
```

Comment:
Which .env file is this? Is it the one authentik reads or one that WUD reads or is that the same file? ;-)